schematics/table/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.ts

```diff
@@ -0,0 +1,132 @@
+import { Component, OnInit, ViewChild, <% if(!!viewEncapsulation) { %>, ViewEncapsulation<% }%><% if(changeDetection !== 'Default') { %>, ChangeDetectionStrategy<% }%> } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { map } from 'rxjs/operators/map';
+import { MatPaginator, MatSort } from '@angular/material';
+import { DataSource } from '@angular/cdk/collections';
```

Comment:
Should we have the spaces around the curly braces here? We don't have them in our source.

Reply:
The default schematics have them so I'd tend to stay the same for consistency. @jelbourn - what do u think?

Reply:
Yeah, we should be consistent with the rest of the CLI even if spaces in braces is nonsense.